### PR TITLE
[enh] Update upstream

### PR DESCRIPTION
**Pull request by zamentur**

### CODE REVIEW

**Comment by nqb** [comment]
Hello @zamentur,

Do you plan to merge this quickly ?

Thanks.

**Comment by nqb** [comment]
`python3-crypto` is necessary on stretch to run this script.

File: scripts/_common.sh [hunk 1]
```diff
@@ -5,7 +5,7 @@
 #=================================================
 
 # dependencies used by the app
-pkg_dependencies="python3-yaml python3-dnspython python3-requests-toolbelt python3-spf whois smartmontools"
+pkg_dependencies="python3-dbus python3-yaml python3-dnspython python3-requests-toolbelt python3-spf whois smartmontools"
```
Comment (alexAubin) [inline scripts/_common.sh]:
```suggestion
pkg_dependencies="python3-crypto python3-dbus python3-yaml python3-dnspython python3-requests-toolbelt python3-spf whois smartmontools"
```

**Comment by alexAubin** [comment]
ˢᵐᵒˡ ᵇᵘᵐᵖ

**Pull request merged**